README.md

```diff
@@ -4,6 +4,8 @@ Welcome! We are very pleased you want to contribute to the Kubernetes documentat
 
 You can click the **Fork** button in the upper-right area of the screen to create a copy of this repository in your GitHub account called a *fork*. Make any changes you want in your fork, and when you are ready to send those changes to us, go to your fork and create a new pull request to let us know about it.
 
+Once your pull request is created, a [Kubernetes reviewer] (https://github.com/kubernetes/kubernetes.github.io/blob/master/OWNERS) will take responsibility for providing clear, actionable feedback.  As the owner of the pull request, **it is your responsibility to modify your pull request to address the feedback that has been provided to you by the Kubernetes reviewer.**  Also note that you may end up having more than one Kubernetes reviewer provide you feedback or you may end up getting feedback from a Kubernetes reviewer that is different than the one originally assigned to provide you feedback. Reviewers will do their best to provide feedback in a timely fashion but response time can vary based on circumstances.
+
```

Comment:
I don't think [Kubernetes reviewer] is the right list to show here. The list of reviewers is bigger than this. I'm not even sure what this list is. Suggestion: Don't give a list at all. Just say "a Kubernetes reviewer."

It's my own opinion that we shouldn't use bold for general emphasis, but apparently this is not covered in the style guide. Probably should discuss in SID Docs.

Suggested tweaks to the text:
As the owner of the pull request, it is your responsibility to modify your pull request to address the feedback that has been provided to you by the Kubernets reviewer. Also note that you might have more than one reviewer, or you might get feedback from a reviewer who is different from the one originally assigned. Reviewers do their best to provide feedback in a timely fashion but response time can vary based on circumstances.